opensalt/opensalt
# Fix .yarnclean file

Fixes issue with `encore --watch` caused by Yarn autoclean removing important files. 

Closes opensalt/opensalt#602

**Test plan (required)**

1. Remove the `node_modules` folder (so Yarn doesn't get confused about what's already there)
2. Run `bin/yarn`
3. Ensure that the folder `node_modules/upath/build` exists and contains stuff
4. Run `bin/encore dev --watch`
5. Ensure that the command runs successfully and doesn't crash


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/opensalt/opensalt/603)
